spec/features/item_registration_spec.rb

```diff
@@ -44,6 +44,7 @@
 
     find("option[value='goobiWF']")
     select 'goobiWF', from: 'workflow_id'
+    select 'Book (ltr)', from: 'content_type'
```

Comment:
we should explicitly select the content type in this test so that if we change the order in the drop-down list, the test still works